packages/web-twig/src/Resources/components/Breadcrumbs/README.md

```diff
@@ -0,0 +1,100 @@
+# Breadcrumbs
+
+This is Twig implementation of the [Breadcrumbs] component.
+
+Basic example usage:
+
+```html
+<Breadcrumbs items="items" />
+```
+
+Without lexer:
+
+```twig
+{% set items = [
+    {
+        title: 'Root',
+        url: '#rootUrl',
+    },
+    {
+        title: 'Category',
+        url: '#categoryUrl',
+    },
+    {
+        title: 'Subcategory',
+        url: '#subcategoryUrl',
+    },
+    {
+        title: 'Current page',
+        url: '#currentUrl',
+    },
+] %}
+
+{% embed "@spirit/breadcrumbs.twig" with { props: {
+    items: items
+}} %}{% endembed %}
+```
+
+Example of custom usage:
+
+```html
+<Breadcrumbs>
```

Comment:
We should be consistent with `web-react`. Custom children are not allowed now in React Breadcrumbs.

Reply:
It is now.